modules/mw.cx.TargetArticle.js

```
@@ -259,9 +259,12 @@ mw.cx.TargetArticle.prototype.publishSuccess = function (response, jqXHR) {
 			this.showErrorCaptcha.bind(this, publishResult.edit.captcha)
 		);
 	}
+	const publish_Result = (publishResult.save_result_all && publishResult.save_result_all.mdwiki_result)
+		? publishResult.save_result_all.mdwiki_result
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)